src/Site.js

```diff
@@ -666,32 +670,77 @@ function getPluginPath(rootPath, plugin) {
   }
 
   // Check in src folder
-  const defaultPath = path.join(__dirname, BUILT_IN_PLUGIN_FOLDER_NAME, `${plugin}.js`);
+  const srcPath = path.join(__dirname, BUILT_IN_PLUGIN_FOLDER_NAME, `${plugin}.js`);
+  if (fs.existsSync(srcPath)) {
+    return srcPath;
+  }
+
+  // Check in default folder
+  const defaultPath = path.join(__dirname, BUILT_IN_PLUGIN_DEFAULT_FOLDER_NAME, `${plugin}.js`);
   if (fs.existsSync(defaultPath)) {
     return defaultPath;
   }
 
   return '';
 }
 
+/**
+ * Finds plugins in the site's default plugin folder
+ */
+function findDefaultPlugins() {
+  const globPath = path.join(__dirname, BUILT_IN_PLUGIN_DEFAULT_FOLDER_NAME);
```

Comment:
This assumes all built-in plugins are default plugins (i.e. turned **on** by default), which is not true.
For example, Algolia search plugin loads unnecessary files in the site if not used, so it should be turned **off** by default.

Also, this is known at MarkBind dev time. Shall we use a constant as a whitelist?

Reply:
Currently there's a separate folder for plugins that are always on (which I'm calling 'default plugins')

```
src/
  plugins/
    default/
      always-on.js
      ... other default plugins
    must-be-turned-on.js
    ... other normal plugins
```

Plugins that should be off can be placed outside the default folder, where they are treated like normal plugins

I think this might be better than using a whitelist as that has to be hardcoded...

Reply:
Ah. `BUILT_IN_PLUGIN_DEFAULT_FOLDER_NAME` → `BUILT_IN_DEFAULT_PLUGIN_FOLDER_NAME`?

Reply:
> Ah. `BUILT_IN_PLUGIN_DEFAULT_FOLDER_NAME` → `BUILT_IN_DEFAULT_PLUGIN_FOLDER_NAME`?

Sounds better :+1:.

Reply:
Sure no problem